react/features/base/conference/functions.ts

```diff
@@ -388,6 +388,24 @@ export function getCurrentConference(stateful: IStateful): IJitsiConference | un
     return joining || passwordRequired || membersOnly;
 }
 
+/**
+ * Returns whether the current conference is a P2P connection.
+ * Will return `false` if it's a JVB one, and `null` if there is no conference.
+ *
+ * @param {IStateful} stateful - The redux store, state, or
+ * {@code getState} function.
+ * @returns {boolean|null}
+ */
+export function isP2pActive(stateful: IStateful): boolean | null {
+    const conference = getCurrentConference(toState(stateful));
+
+    if (!conference) {
+        return null;
```

Comment:
I kind of used [this](https://github.com/jitsi/lib-jitsi-meet/blob/0748d89a991de6cecb74e679c1cb4afff5e59c80/JitsiConference.js#L2648) as a reference and thought it would matter to tell the difference between jvb and no conference

Reply:
They should be able to infer it from the fact that there is just 1 participant. Bull is still falsey, so no strong opinion.

Reply:
The event will return null for that case, so I'd like to keep them consistent if that's ok